packages/devtools/src/commands/performActions.ts

```diff
@@ -180,6 +183,27 @@ export default async function performActions(
             continue
         }
 
+        if (action.type === 'wheel') {
+            for (const singleAction of action.actions) {
+                if (singleAction.origin) {
+                    const windowSize = await getWindowRect.call(this)
+                    const location = await getElementRect.call(this, { elementId: singleAction.origin[ELEMENT_KEY] })
+                    await page.mouse.wheel({
+                        deltaX: location.x,
+                        deltaY: location.y - windowSize.height
+                    })
+                } else if (singleAction.x || singleAction.y) {
+                    await page.mouse.move(singleAction.x || 0, singleAction.y || 0)
+                }
+
+                await page.mouse.wheel({
```

Comment:
This always get's executed, it feels like we should only scroll once when an origin is assigned. Is this correct?

Reply:
Good catch, I updated the implementation as it had an error anyway. I hope the updated version is more clear.